code/lisibilite/Lisibilite.py

```diff
@@ -19,27 +23,22 @@ def __init__(self, filename: str = None, contents: str = None):
 
         try:
             if filename:
-                self.metrics = self.__computeMetricsFromFile(filename)
-                self.computed = True
+                self.outputModel = self.__computeMetricsFromFile(filename)
             elif contents:
-                self.metrics = self.__computeMetricsFromString(contents)
-                self.computed = True
+                self.outputModel = self.__computeMetricsFromString(contents)
             else:
                 LOG.error('Bad input. One of filename or contents should be set')
                 raise BadInputError('Bad input. One of filename or contents should be set')
         except Exception as error:
             LOG.error(f'Error in computing the lisibilite errors. Error = {error}')
             raise IOError(f'Error in computing the lisibilite errors. Error = {error}')
 
-    def __computeMetricsFromString(self, contents: str) -> CoreMetrics:
+    def getReadabilityMetrics(self) -> OutputDataModel:
         """
-        Private: Method to compute the metrics from an input string
-        :param contents: The string for which the metrics should be computed
-        :return: Metrics object
+        The method to retrieve the Output Model
+        :return: The output model
         """
-        LOG.debug("Computing metrics from string")
-        metrics = CoreMetrics()
-        return metrics
+        return self.outputModel
```

Comment:
Should this be initialized to `None` in `init` ?
Or can use the computed flag is needed?

Reply:
Done